src/main/java/core/basesyntax/Exception/NoValidDataException.java

```diff
@@ -0,0 +1,7 @@
+package core.basesyntax.Exception;
```

Comment:
The package name should be in lowercase according to Java naming conventions. Consider renaming 'Exception' to 'exception'.